Sources/App/routes.swift

```diff
@@ -19,15 +19,15 @@ func routes(_ app: Application) throws {
 
     let shoppingCart = app.grouped("shoppingCart")
     /// Returns single shopping cart
-    shoppingCart.get(use: shoppingCartController.read)
+    app.get("shoppingCart", use: shoppingCartController.read)
```

Comment:
I tested and it works. But just curious why you are not using  `shoppingCart.get` but `app.get` instead here